mlflow/metrics/genai/metric_definitions.py

```diff
@@ -67,3 +67,60 @@ def correctness(
         aggregations=["mean", "variance", "p90"],
         greater_is_better=True,
     )
+
+
+def relevance(
+    model: Optional[str] = None,
+    metric_version: Optional[str] = _get_latest_metric_version(),
```

Comment:
can make this `None` in the function signature and set the value in L110 with a call to `_get_latest_metric_version` like we do for examples and model? this pattern is more idiomatic.